pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -165,6 +157,7 @@ public RpcApi convert(final String name) throws RpcApisConversionException {
   // This seems to be the only way it works with Picocli.
   // Also many other software use the same negative option scheme for false defaults
   // meaning that it's probably the right way to handle disabling options.
+  // TODO change naming and find a way to make it able to define true or false
```

Comment:
Better to leave this TODO in a jira ticket or somewhere rather than in the code.

Reply:
it's just a reminder for myself during the refactoring process. Will be removed.